pkg/policy/opa/engine.go

```diff
@@ -465,3 +469,21 @@ func (e *Engine) filterSkipRules(policyPath string, skipRules []string) {
 		}
 	}
 }
+
+func (e *Engine) filterBySeverity(policyPath string, severity string) {
+
+	// temporary map to store data from original rego data map
+	tempMap := make(map[string]*RegoData)
+	for ruleID, regoData := range e.regoDataMap {
+
+		if utils.CheckSeverity(regoData.Metadata.Severity, severity) {
+			tempMap[ruleID] = regoData
+		}
+	}
+	if len(tempMap) == 0 {
+		zap.S().Warnf("policy path: %s, doesn't have any rule matching the severity level : %s", policyPath, severity)
```

Comment:
this log should not be at warn level, nothing bad has happened.

Reply:
I did it following the filterSkipRules and filterScanRules methods. Doesn't hurt to inform the user :D

Reply:
yeah, informing or warning is the difference :D

Reply:
agree with @patilpankaj212, do think there is a need for a warning, may be a `debug` level log